migrate-profile.md

```diff
@@ -0,0 +1,54 @@
+# Migrate Tanzu Application Platform Profiles
+
+This document describes how to migrate from one Tanzu Application Platform profile to another.
+
+You can perform fresh install of Tanzu Application Platform 1.1 by following the instructions in [Installing Tanzu Application Platform](install-intro.md).
+
+## <a id='prereqs'></a> Prerequisites
+
+Before you migrate from one Tanzu Application Platform to another:
```

Comment:
Suggest to add clarity here, even thou the title of the doc is for migrating profiles, this line reads you can migrate all of TAP.